src/main/webapp/app/course/dashboards/assessment-dashboard/assessment-dashboard.component.html

```diff
@@ -206,9 +206,6 @@ <h4 jhiTranslate="artemisApp.assessmentDashboard.exerciseTable.title"></h4>
                                         [togglingSecondCorrectionButton]="isTogglingSecondCorrection.get(exercise.id!)!"
                                     ></jhi-second-correction-enable-button>
                                 </ng-container>
-                                <a class="btn btn-primary btn-sm mb-1" *ngIf="course?.isAtLeastInstructor" [routerLink]="getSubmissionsLinkForExercise(exercise)">
```

Comment:
If it has the css class btn, looks and works like a button; it probably should be a `<button>`